physics/photochem/module_h2ophys.F90

```diff
@@ -228,6 +231,9 @@ subroutine run(this, dt, p, h2opltc, h2o)
              h2o(iCol,iLev)  = (h2oib(iCol) + (pltc(iCol,1)+pltc(iCol,3)*temp)*dt) / (1.0 + temp*dt)
           endif
        enddo
+
+       if (present(dqv_dt_prd)) dqv_dt_prd(:, iLev) = pltc(:, 1) * dt
+       if (present(dqv_dt_qv))  dqv_dt_qv(:, iLev)  = (h2o(:, iLev) - pltc(:, 3)) * dt / pltc(:, 2)
```

Comment:
@DWesl In debug mode tests, this line is leading to a runtime failure due to division by zero.

Reply:
The code above does something like the below, I'm running regression tests on this variant now.
```suggestion
       if (present(dqv_dt_qv))  then
          where (pltc(:, 2) == 0)
             dqv_dt_qv(:, iLev)  = (h2o(:, iLev) - pltc(:, 3)) * dt / pltc(:, 2)
          elsewhere
             dqv_dt_qv(:, iLev) = 0
          end where
       end if
```

Reply:
Right, the `where` condition should do the division only if the denominator is *not* zero.  Edited above.